lib/dns.js

```diff
@@ -0,0 +1,51 @@
+const LRUCache = require('lru-cache')
+const dns = require('dns')
+
+const defaultOptions = exports.defaultOptions = {
+  family: undefined,
+  hints: dns.ADDRCONFIG,
+  all: false,
+  verbatim: true,
+}
+
+const lookupCache = exports.lookupCache = new LRUCache({ max: 50 })
```

Comment:
I don't have any input on what this number should be, but I'm curious on what factors would lead us to change this. Do we have a target package.json that would install without evicting any DNS cache entries?

Reply:
this is per-host, so unless you have more than 50 hosts you're making requests to in a single package-lock this should be just fine. also the lru-cache behaviors mean even if you did unless there's an even distribution you'll end up hitting the cache for the majority of requests anyway.